src/js/modules/setup/configs/setup-sate/setup-state.config.js

```diff
@@ -12,65 +12,7 @@
                 controller: "SetupCtrl",
                 templateUrl: "js/modules/setup/controllers/setup/setup.tpl.html",
                 resolve: {
-                    /**
-                     * check for extra languages to enable
-                     * if new language is new preferred, set it
-                     */
-                    preferredLanguage: function(CONFIG, $rootScope, settingsService, blocktrailLocalisation, launchService, AppVersionService) {
-                        return launchService
-                            .getWalletConfig()
-                            .then(function(result) {
-                                AppVersionService.checkVersion(
```

Comment:
this `AppVersionService.checkVersion` was removed, plz bring it back

Reply:
Returned